wuvt/static/css/wuvt.css

```diff
@@ -128,26 +128,32 @@ header #mainheader h1 {
     margin: 20px 0px 0px 0px;
     font-size: 32px;
     text-indent: -9999px;
+    vertical-align: top;
+    display: inline-block;
```

Comment:
this would probably be an appropriate place to edit the PR yourself, @jbellerb

Reply:
![image](https://github.com/wuvt/wuvt-site/assets/95111563/7284c4ac-a541-4529-9446-ae29749f5206)
Header looks like this using the new code

Reply:
Sorry for being super late, I'm still not used to checking Outlook. I communicated this extremely poorly but that snippet was intended more as an example for what CSS can do rather than a request to copy what I had written. Either way, the snippet worked fine when I copied it over just now (it was a set of changes I made with the dev tools and I never actually tested it so I'm kind of surprised) and I can't replicate your screenshot. I'd be happy to talk more about it on Slack.